pylsp/config/source.py

```diff
@@ -69,6 +69,8 @@ def _get_opt(config, key, option, opt_type):
 
 
 def _parse_list_opt(string):
+    if string.startswith("\n"):
+        return [s.strip().rstrip(",") for s in string.split("\n") if s.strip()]
```

Comment:
This seems very specific to flake8. Could it affect reading config files for oher plugins?

Reply:
I just added a commit that restricts this customization to flake8 only.

Reply:
N.B.: Ideally, this library would use the `flake8` library itself to parse these options (e.g. [`parse_files_to_codes_mapping`](https://github.com/PyCQA/flake8/blob/7a371265cf876736b8d03907df26b813874b4201/src/flake8/utils.py#L89)).